### ISSUE TITLE
Not possible to hide tooltip when event === hover and wrapperOptions.position === true

### ISSUE THREAD

**Issue by kartamyshev** [gilbarbara/react-floater#33]
>Expected behavior
Tooltip should be hidden when mouseleave from one and mouseenter to another tooltip

>Actual behavior
Tooltip is not hidden. Floater innerHTML (in this case `<span>?</span>`) disappeared.

>Steps to reproduce the problem

Here is the code
```
const FloaterComponent = ({ target }) => {
  return <div>
    <Floater
      target={target}
      event="hover"
      eventDelay={1}
      wrapperOptions={{ position: true }}
      content="This is Floater Content">
      <span>?</span>
    </Floater>
  </div>;
};

const Floaters = () => {
  return <div>
    <FloaterComponent target="#one" />
    <FloaterComponent target="#two" />
    <FloaterComponent target="#three" />
    <FloaterComponent target="#four" />
  </div>;
};

ReactDOM.render(
  <div className="wrapper">
    <div id="one">One</div>
    <div id="two">Two</div>
    <div id="three">Three</div>
    <div id="four">Four</div>
    <Floaters />
  </div>,
  document.getElementById('application-container')
);
```

>React version
16.3.1

>React-Tooltips version
0.5.2

>Error stack

**Comment by gilbarbara** [comment]
Please create an example at https://codesandbox.io

**Comment by kartamyshev** [comment]
Sure. Here it is https://x99rp8ry1q.codesandbox.io/

**Comment by gilbarbara** [comment]
Using `event` with hover the positioned wrapper hides when the floater is visible and there's no mouseLeave to perform anymore.
This will never work and I'll add a note in the README about this limitation.

The close button wasn't working also, which I fixed in 0.5.3

Thanks
